.changeset/twelve-hounds-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade-old": patch
+---
+
+on keyboard press and text align props addition in TextInput. Text Align can take values left, right and center. On Keyboard Press will take native event as argument. These props are optional
```

Comment:
It's `onKeyboardPress` or `onKeyPress`? Seems to be `onKeyPress` in https://github.com/razorpay/blade/pull/435 PR

Reply:
https://github.com/razorpay/blade/blob/c3be4be2085659412a84d2edea59f62c7caf2b22/packages/blade-old/src/atoms/TextInput/TextInput.native.js#L504

Reply:
```suggestion
`onKeyPress` and `textAlign` props addition in `TextInput`. `textAlign` can take values `left`, `right`, and `center`. `onKeyPress` will take native event as argument. These props are optional
```